Packs/Phishing/Scripts/PhishingDedupPreprocessingRule/PhishingDedupPreprocessingRule.py

```diff
@@ -131,6 +149,8 @@ def preprocess_incidents_df(existing_incidents):
 def filter_out_new_incident(existing_incidents_df, new_incident):
     same_id_mask = existing_incidents_df['id'] == new_incident['id']
     existing_incidents_df = existing_incidents_df[~same_id_mask]
+    earlier_incidents_mask = existing_incidents_df['id'].astype(int) < int(new_incident['id'])
```

Comment:
you need to consider that the ID of the incidents gonna change because we are moving to ES (elasticsearch).
Wrap with try-catch covenanting the id to int.
When you sort use created & id column - it will always return same incident as first.
if there is a few with the same time -it will just get the id with smaller id (if you success parse to int it will the most early ingested one)
when we move to ES ids not gonna be INT anymore... :(